client/styles/App.module.scss

```diff
@@ -1,4 +1,5 @@
 @use "variables.module.scss" as variables;
+@use 'login.scss';
```

Comment:
I don't think line 2 is necessary. I would recommend renaming the file to match the name of the page or component that it corresponds to. In this case, LoginForm.module.scss. Then import that into the LoginForm.tsx file.

Reply:
Agreed. 'Component.module.scss' or 'PageName.module.scss' should be the name of all scss files, that are not variables, globals, or others.